src/tainting/Taint.ml

```diff
@@ -65,7 +66,7 @@ let rec _show_call_trace show_thing = function
 (* Signatures *)
 (*****************************************************************************)
 
-type arg_pos = string * int [@@deriving show]
+type arg_pos = string * int [@@deriving show, compare]
```

Comment:
Is that any better than polymorphic compare? It's probably faster, and maybe automatically ignores functions instead of raising an exception. But if we had used `ppx_compare` previously and then changed the data type as we did, would have that gave us any warning that we needed to write a custom `compare` for the `offset`s?

Reply:
I expect it would have complained that it didn't have a `compare` function for `offset`, right?